t/admin/consumer-group.t

```diff
@@ -74,11 +74,6 @@ __DATA__
             ngx.status = code
             ngx.say(body)
 
-            local res = assert(etcd.get('/consumer_groups/company_a'))
-            local create_time = res.body.node.value.create_time
-            assert(create_time ~= nil, "create_time is nil")
-            local update_time = res.body.node.value.update_time
-            assert(update_time ~= nil, "update_time is nil")
```

Comment:
Why do you remove this, the `create_time` and `update_time` don't come from the input, but they still exist

Reply:
done!

Reply:
> Why do you remove this, the `create_time` and `update_time` don't come from the input, but they still exist

Test github action.